examples/make_assets/ptable/ptable_matplotlib.py

```diff
@@ -69,7 +69,9 @@
     cbar_axis="x",
     color_elem_strategy="background",
     add_elem_type_legend=True,
-    # x_range=(0, None),
+    symbol_pos=(0.25, 0.8),
+    anno_text={elem.symbol: str(idx - 1) for idx, elem in enumerate(Element)},
```

Comment:
Noticed a funny behavior: 
```python
from pymatgen.core.periodic_table import Element


for idx, ele in enumerate(Element):
    print(idx, ele.symbol)

    if idx > 3:
        break

>>>     0 H
        1 H
        2 H
        3 He
        4 L

print(f"{Element.D.symbol=}")  >>> Element.D.symbol='H'
```

I believe [this is intended to return isotope symbols as its most common one](https://github.com/materialsproject/pymatgen/blob/ea7c3390e2db99a35f3eee242073ffc2830bbd4c/src/pymatgen/core/periodic_table.py#L136-L145).

Then [not sure why define H/D/T separately when they symbol all map to `H`](https://github.com/materialsproject/pymatgen/blob/ea7c3390e2db99a35f3eee242073ffc2830bbd4c/src/pymatgen/core/periodic_table.py#L866-L877).